core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1537,6 +1542,26 @@ class EMConfig {
     @Cfg("Specify a path to save all executed sql commands to a file (default is 'sql.txt')")
     var saveExecutedSQLToFile : String = "sql.txt"
 
+    @Experimental
+    @Cfg("Whether to enable extra targets for responses, e.g., regarding nullable response, having extra targets for whether it is null")
+    var enableRPCExtraResponseTargets = false
+
+    @Experimental
+    @Cfg("Whether to enable customized responses indicating business logic")
+    var enableRPCCustomizedResponseTargets = false
+
+    @Experimental
+    @Cfg("Whether to generate RPC tests which is independent from EM driver.")
+    var enablePureRPCTestGeneration = false
+
+    @Experimental
+    @Cfg("Whether to generate RPC Assertions based on instance")
+    var enableRPCAssertionWithInstance = false
```

Comment:
what does it mean "based on instance"?

Reply:
update it to `Whether to generate RPC Assertions based on response instance`